indi-eqmod/skywatcher.cpp

```diff
@@ -1723,7 +1723,13 @@ bool Skywatcher::dispatch_command(SkywatcherCommand cmd, SkywatcherAxis axis, ch
         try
         {
             if (read_eqmod())
+            {
+                if (i > 0)
+                {
+                    LOGF_WARN("%s() : serial port read failed for %dms (%d retries), assess cable quality.", __FUNCTION__, (i*EQMOD_TIMEOUT)/1000, i);
```

Comment:
It's not necessarily wired. It could be wired, bluetooth, or WiFi.

Reply:
Changed text.